ads/google/doubleclick.js

```diff
@@ -100,14 +100,12 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     const primaryWidth = primarySize[0];
     const primaryHeight = primarySize[1];
     // By default, this flag is true.
-    const multiSizeValidation = data.multiSizeValidation ?
-        data.multiSizeValidation == 'true' : true;
 
     getMultiSizeDimensions(
         multiSizeDataStr,
         primaryWidth,
         primaryHeight,
-        multiSizeValidation,
+        (data.multiSizeValidation || 'true') == 'true',
```

Comment:
hmm that's irritating, I was hoping you'd simply be able to check that `undefined != 'false'` and that would work but I guess not. Alright that's fine then. My nit if it has to stay this way would be to change this line to
`(data.multiSizeValidation == 'true') || true`